medusa/restore_cluster.py

```diff
@@ -126,9 +126,9 @@ def __init__(self, cluster_backup, config, temp_dir, host_list, seed_target, kee
         self.fqdn_resolver = HostnameResolver(fqdn_resolver)
         self._version_target = version_target
 
-    def execute(self):
+    def prepare_restore(self):
         logging.info('Ensuring the backup is found and is complete')
-        if not self.cluster_backup.is_complete():
+        if not self.config.kubernetes.enabled and not self.cluster_backup.is_complete():
```

Comment:
> If k8s is not enabled, should we provide that message so the user knows that the k8s enablement is required?

I don't think so. This condition was introduced because due to hostname resolving issues, backups that are complete may not be seen as such in kubernetes deployments.

Lines 135 to 140 handle both non-remote and remote restores. It's just that you have two ways to trigger a remote restore (in Medusa's orchestration) which is either to provide a seed target and let Medusa do the mapping, or provide a file with the mapping.